db/seeds.d/106-mail_notifications.rb

```diff
@@ -30,7 +30,7 @@
 ]
 
 notifications.each do |notification|
-  ::MailNotification.find_or_create_by_name(notification)
+  ::MailNotification.create(notifications) unless ::MailNotification.where(:name => notification[:name]).exists?
```

Comment:
Talked with you on IRC but I think this would be better:

```ruby
::MailNotification.where(name: notification[:name]).first_or_create!(notification)
```